content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,132 @@
+---
+title:       "Newsletter 103 - Late 2023 news"
+author:      "GeoB99"
+date:        2023-08-06
+tags:        [ newsletter ]
+---
+
+Salutations from the ReactOS project team!
+On the previous newsletter we talked about the overall ReactOS releasing progress scheme and development status of the project, as well as the hiring of our long-term developer Hermès Bélusca-Maïto (HBelusca).
+We are attempting to make an effort on publishing regularly at least 3 newsletters per year, depending on how the development workflow goes.
+In this newsletter we will highlight some of the contributions by project developers and contributors alike as well as future plans and headlines.
+
+## 0.4.15 release status
+On [102nd newsletter](/blogs/newsletter-102/) we talked about the ReactOS release situation and how we are tackling it.
+Despite our on-going effort and continuation for the project's development, many people have asked about what happened to 0.4.15 and if there will be such a release.
+The answer to these questions is yes -- there will be 0.4.15 and it will be released. The 0.4.15 release has been long delayed due to outstanding regressions and bugs.
```

Comment:
```suggestion
The answer to these questions is yes -- there will be 0.4.15 and it will be released.
The 0.4.15 release has been long delayed due to outstanding regressions and bugs.
```